tests/yam/agama/patch_agama_tests.pm

```diff
@@ -9,11 +9,19 @@ use base Yam::Agama::patch_agama_base;
 use strict;
 use warnings;
 use testapi qw(assert_script_run get_required_var select_console);
+use Utils::Architectures;
+use version_utils qw(is_sle);
 
 sub run {
     select_console 'root-console';
     my ($repo, $branch) = split /#/, get_required_var('YUPDATE_GIT');
     assert_script_run("yupdate patch $repo $branch", timeout => 60);
+
+    if (is_sle() && is_s390x()) {
+        assert_script_run("curl --insecure -o /tmp/ca-certificates-suse.rpm https://download.suse.de/browse/ibs/SUSE:/CA/SLE_15_SP7/noarch/ca-certificates-suse-1.0-150700.8.1.noarch.rpm");
+        assert_script_run("zypper -n install /tmp/ca-certificates-suse.rpm");
+        assert_script_run("systemctl restart agama");
```

Comment:
this workaround is not related with patching agama with our tests, please add in `agama.pm` with a record_softfailure pointing to a new bug that we should open.

Reply:
029a14f46